great-idea-website/index.html

```diff
@@ -16,48 +16,75 @@
 
   <body>
 
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
+    <div class="container">
+      <header>
+      <nav>
+<a href="#">Services</a>
+<a href="#">Product</a>
+<a href="#">Vision</a>
+<a href="#">Features</a>
+<a href="#">About</a>
+<a href="#">Contact</a>
+          </nav>
   <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
+  </header>
+  <section class="intro">
+  <div class="introtext">
+    <h1>Innovation</h1>
+  <h1>On</h1>
+  <h1>Demand</h1>
+  <p>Get Started</p>
+</div>
+<div class="circleimg">
   <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
+  </div>
+  </br>
+</br>
+</br>
```

Comment:
<br> tag is great to use for example when you need a line break in the text, and when you want to add some space around elements I would play with margin and the padding property